diff --git a/config.def.h b/config.def.h
index 047b5d1..0cdc282 100644
--- a/config.def.h
+++ b/config.def.h
@@ -1,6 +1,6 @@
 
-#define FONT "-*-*-medium-r-*-*-*-120-75-75-*-60-*-*"
-#define BOLDFONT "-*-*-bold-r-*-*-*-120-75-75-*-60-*-*"
+#define FONT "-*-terminus-medium-*-*-*-16-*-*-*-*-*-iso10646-*"
+#define BOLDFONT "-*-terminus-bold-*-*-*-16-*-*-*-*-*-iso10646-*"
 
 /* Space in pixels around the terminal buffer */
 #define BORDER 2
